sorc/ncep_post.fd/MISCLN.f

```diff
@@ -1105,136 +1102,6 @@ SUBROUTINE MISCLN
               ENDIF
             ENDIF
 !
-!  ADD FD LEVEL DUST/ASH (GOCART)
```

Comment:
@zhanglikate You remove AERFD output (601-605) in this routine. Do you think the lines 839 to 853 can be removed too?
 ```
           IF (IGET(601)>0) THEN
            IF (LVLS(IFD,IGET(601))>1) ITYPEFDLVL(IFD)=2
           ENDIF
           IF (IGET(602)>0) THEN
            IF (LVLS(IFD,IGET(602))>1) ITYPEFDLVL(IFD)=2
           ENDIF
           IF (IGET(603)>0) THEN
            IF (LVLS(IFD,IGET(603))>1) ITYPEFDLVL(IFD)=2
           ENDIF
           IF (IGET(604)>0) THEN
            IF (LVLS(IFD,IGET(604))>1) ITYPEFDLVL(IFD)=2
           ENDIF
           IF (IGET(605)>0) THEN
            IF (LVLS(IFD,IGET(605))>1) ITYPEFDLVL(IFD)=2
           ENDIF
```

Reply:
@WenMeng-NOAA  Yes. Updated.